runtime/lua/vim/lsp/buf.lua

```diff
@@ -879,7 +872,7 @@ local hierarchy_methods = {
 local function hierarchy(method)
   local kind = hierarchy_methods[method]
   if not kind then
-    error('unsupported method ' .. method)
+    vim.notify(lsp._unsupported_method(method), vim.log.levels.WARN)
```

Comment:
is this intentionally changing from error => warn level ?

Reply:
Yeah because I see no reason to `error` here instead of logging/notifying as we do everywhere else.

Reply:
I'm pretty sure this needs to remain an error as it's essentially an assertion on the input and can only trigger to do incorrect internal usage. This is also missing a `return`.

Reply:
Alright I’ll revert this soon (AFK rn).